.rust_alpha/polly/synthesize-speech/src/main.rs

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+use std::fs;
+use std::process;
+
+use polly::model::{OutputFormat, VoiceId};
+use polly::{Client, Config, Region};
+
+use aws_types::region::{ProvideRegion};
+
+use bytes::Buf;
+use structopt::StructOpt;
+use tokio::io::AsyncWriteExt;
+use tracing_subscriber::fmt::format::FmtSpan;
+use tracing_subscriber::fmt::SubscriberBuilder;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// The region
+    #[structopt(short, long)]
+    default_region: Option<String>,
+
+    /// The file containing the text to synthesize
+    #[structopt(short, long)]
+    filename: String,
+
```

Comment:
periods for comments, here and throughout

Reply:
Fixed